crates/wadm/src/server/handlers.rs

```diff
@@ -248,13 +248,20 @@ impl<P: Publisher> Handler<P> {
         };
 
         for model in &mut data {
+            // TODO: we want to disregard the status if the lattice isn't being observed yet... how?
+            // Need some form of query to check if the lattice is being observed
+            #[allow(deprecated)]
```

Comment:
This TODO is why #346 isn't solved here, we need an indicator to know that a lattice isn't observed yet in order to _truly_ decide whether to take the last status or put in waiting. Was hoping to noodle with @thomastaylor312 on this one, but it doesn't need to be fixed in this PR.